packages/ui/Image/Image.js

```diff
@@ -132,12 +124,22 @@ const ImageBase = kind({
 		 * @default 'fill'
 		 * @public
 		 */
-		sizing: PropTypes.oneOf(['fit', 'fill', 'none'])
+		sizing: PropTypes.oneOf(['fit', 'fill', 'none']),
+
+		/**
+		 * String value or Object of values used to determine which image will appear on
+		 * a specific screenSize.
+		 *
+		 * @type {String|Object}
+		 * @public
+		 */
+		src: PropTypes.oneOfType([PropTypes.string, PropTypes.object])
 	},
 
 	defaultProps: {
 		placeholder: '',
-		sizing: 'fill'
+		sizing: 'fill',
+		src: ''
```

Comment:
Do we need to specify a default here?  And, if so, should we document it in the prop?